src/bikeracks/helpers.py

```diff
@@ -151,90 +130,58 @@ def get_vote_status(database, rack_id, user_id):
-    try:
-        query = "UPDATE bikeracks SET upvote_count = upvote_count + 1 WHERE rack_id=?"
-        database.execute(query, (rack_id,))
-        database.commit()
-    except sqlite3.Error as e:
-        print("Database error:", e)
-        return "", 500
-    except KeyError as key_e:
-        print("Error with key: {}".format(key_e))
-        return "", 500
+    query = "UPDATE bikeracks SET upvote_count = upvote_count + 1 WHERE rack_id=?"
+    database.execute(query, (rack_id,))
+    database.commit()
     
     return
     
 def increment_downvote_count(rack_id, database):
-    try:
-        query = "UPDATE bikeracks SET downvote_count = downvote_count + 1 WHERE rack_id=?"
-        database.execute(query, (rack_id,))
-        database.commit()
-    except sqlite3.Error as e:
-        print("Database error:", e)
-        return "", 500
-    except KeyError as key_e:
-        print("Error with key: {}".format(key_e))
-        return "", 500
-    
+
+    query = "UPDATE bikeracks SET downvote_count = downvote_count + 1 WHERE rack_id=?"
+    database.execute(query, (rack_id,))
+    database.commit()
+
     return
     
 def get_count_percentage(rack_id, database):
     # return the percentage of downvote_count for a rack
-    try:
-        query = "SELECT downvote_count FROM bikeracks WHERE rack_id=?"
-        downvote_count = database.execute(query, (rack_id,)).fetchone()
-        downvote_count = dict_from_row(downvote_count)
-        
-        query = "SElECT upvote_count FROM bikeracks WHERE rack_id=?"
-        upvote_count = database.execute(query, (rack_id,)).fetchone()
-        upvote_count = dict_from_row(upvote_count)
-        
-        
-        upvote_count = upvote_count['upvote_count']
-        downvote_count = downvote_count['downvote_count']
-        total_votes = upvote_count + downvote_count
-        
-        downvote_percentage = floor((downvote_count / total_votes) * 100)
-        upvote_percentage = floor((upvote_count / total_votes)*100)
-
-        return {'downvote_percentage': downvote_percentage, 'upvote_percentage': upvote_percentage}
-    except sqlite3.Error as e:
-        print("Database error:", e)
-        return "", 500
-    except KeyError as key_e:
-        print("Error with key: {}".format(key_e))
-        return "", 500
+
+    query = "SELECT downvote_count FROM bikeracks WHERE rack_id=?"
+    downvote_count = database.execute(query, (rack_id,)).fetchone()
+    downvote_count = dict_from_row(downvote_count)
+    
+    query = "SElECT upvote_count FROM bikeracks WHERE rack_id=?"
+    upvote_count = database.execute(query, (rack_id,)).fetchone()
+    upvote_count = dict_from_row(upvote_count)
+    
+    
+    upvote_count = upvote_count['upvote_count']
+    downvote_count = downvote_count['downvote_count']
+    total_votes = upvote_count + downvote_count
+    
+    downvote_percentage = floor((downvote_count / total_votes) * 100)
+    upvote_percentage = floor((upvote_count / total_votes)*100)
```

Comment:
This isn't wrong, but it's going to lead to situations where the two don't add up to 100 due to rounding.

```python
total = counts['upvote_count'] + counts['downvote_count']
# Favor upvotes by floor-ing downvote instead
downvote = floor(downvote_count / total) * 100
upvote = 100 - downvote
```

Reply:
Changed it.